mypy/server/deps.py

```diff
@@ -150,7 +150,6 @@ def __init__(self,
     # TODO (incomplete):
     #   from m import *
     #   await
-    #   named tuples
```

Comment:
If you are removing the TODO item also from here, then it is better to be sure that Python 3.6 class syntax for named tuples also works.